lib/chef/provider/aws_auto_scaling_group.rb

```diff
@@ -6,7 +6,7 @@ class Chef::Provider::AwsAutoScalingGroup < Chef::Provisioning::AWSDriver::AWSPr
   protected
 
   def create_aws_object
-    converge_by "create new Auto Scaling Group #{new_resource.name} in #{region}" do
+    converge_by "create Auto Scaling group #{new_resource.name} in #{region}" do
```

Comment:
I love the consistency of changing all the strings to match. I think we should use capitalization whenever the message directly references something in the AWS console. For example, here it should be 'Auto Scaling Group' as capitalized.